models/cleanup.go

```diff
@@ -0,0 +1,118 @@
+package models
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/netlify/gotrue/storage"
```

Comment:
Should this be changed with our rename?
```suggestion
	"github.com/supabase/internal/api/gotrue/storage"
```